mempool/aggregator.go

```diff
@@ -0,0 +1,78 @@
+package mempool
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/libp2p/go-libp2p-core/host"
+)
+
+// TODO: add String()
+type namespace int
```

Comment:
We should make a type outside of consensus and mempool pkgs. Something like `chain.ID` or `concord.ID` is used here. Namespace is the concept of NMT, while we will operate with chains or concords.

Reply:
Also, don't get why is it private